_common/README

```diff
@@ -0,0 +1,38 @@
+<!-- README Template for ory/examples. -->
+<!-- Use the following template for all READMEs in ory/examples. -->
+
+# Title 
+
+<!--Add one sentence describing the project, usecase and key services used. -->
+<!--Add a link to post on the Ory Blog as well as other platforms (dev.to, ...). -->
+
+## Overview
+
+<!-- Optional: Add an diagram if using multiple services or a more detailed explanation than in description or links to other relevant resources. -->
+
+### Prerequisites
+
+<!--Add a list of software needed to run the example, ideally with links to the installation page or installation (e.g. brew) commands. -->
+
+## Develop
+
+<!-- Add a numbered list of commands needed to run the example locally or with Ory Cloud playground. -->
+
+### Environmental Variables
+
+<!-- Optional: If needed add instructions on how to use environmental variables. -->
+
+### Run tests
+
+<!-- Add a numbered list of commands needed to run tests locally. -->
+
+## Deploy
+
+<!-- Optional: Add a list of steps needed to use the example with Ory services selfhosted.-->
+<!-- Optional: Add a list of steps needed to use the example with Ory Cloud developer project.-->
+
+## Contribute
+
+Feel free to [open a discussion](https://github.com/ory/examples/discussions/new) to provide feedback or talk about ideas, or [open an issue](https://github.com/ory/examples/issues/new) if you want to add your example to the repository or encounter a bug.
```

Comment:
I have seen standards to wrap text at 80 characters elsewhere at Ory. Does this standard also apply to this repo?

Reply:
you are right, we still need to apply the standard templates to this repo and add the prettier styles
https://github.com/ory/meta/blob/master/package.json
https://github.com/ory/meta

I think we can commit this though and then later on format everything.